src/compiler/transformer.ts

```diff
@@ -315,16 +353,12 @@ namespace ts {
                 }
 
                 if (lexicalEnvironmentVariableDeclarations) {
-                    const statement = createVariableStatement(
-                        /*modifiers*/ undefined,
-                        createVariableDeclarationList(lexicalEnvironmentVariableDeclarations)
-                    );
-
-                    if (!statements) {
-                        statements = [statement];
-                    }
-                    else {
-                        statements.push(statement);
+                    const groups = groupVariableDeclarationsByLeadingCommentPresence(lexicalEnvironmentVariableDeclarations);

```

Comment:
Do you need to group them (possibly changing the original order), or would `spanMap` (in _core.ts_) be sufficient?